archive/j/javascript/even-odd.js

```diff
@@ -0,0 +1,6 @@
+const input = Number(process.argv[2]); //coerce the input into a number if possible
+if(!Number.isInteger(input)){ //if there is no input, input = undefined and the statement still prints
```

Comment:
Looks like this condition misses the empty input condition ("").

Reply:
You're right. I have added a conditional statement to catch that. I've tested that it catches null, undefined, floating point numbers, and strings with characters in them.